src/components/Proposals/FeedbackModule.js

```diff
@@ -17,10 +18,7 @@ function FeedbackModule({ vote, connectedAccount }) {
 
   let mode = null
 
-  if (
-    vote.challenge &&
-    addressesEqual(vote.challenge.challenger, connectedAccount)
-  ) {
+  if (vote.challenger && addressesEqual(vote.challenger, connectedAccount)) {
```

Comment:
Should we raise this logic up as described [here](https://github.com/aragon/network-dashboard/pull/9/files#r474687155)?